Code/Tools/AssetProcessor/native/tests/ApplicationManagerTests.h

```diff
@@ -9,7 +9,9 @@
 #pragma once
 
 #include <utilities/BatchApplicationManager.h>
+#if !defined(Q_MOC_RUN)
```

Comment:
Why are these UnitTest files being run through QMoc?

Reply:
`Code/Tools/AssetProcessor/native/tests/ApplicationManagerTests.cpp` includes `MockAssetProcessorManager.h`, which contains a `Q_OBJECT`, so `moc` must run on it. `ApplicationManagerTests.cpp` also includes `ApplicationManagerTests.h`, which includes a bunch of stuff that eventually contains a raw string literal, which [moc does not support](https://bugreports.qt.io/browse/QTBUG-55962).